src/chai.js

```diff
@@ -1,17 +1,16 @@
-import chai from 'chai';
+// @flow
+import chai from 'chai'; // eslint-disable-line
```

Comment:
can we always disable a specific eslint rule? what is this disabling?

Reply:
I tried. However, two rules were failing on the same line, lol ☹️

Reply:
can't you just space-separate rules to disable?

Reply:
Ah ha! Just needed a comma. Thanks!
```js
// eslint-disable-line goodeggs/import-order, goodeggs/import-default
```